examples/example.site.yml

```diff
@@ -182,6 +184,9 @@
 #     options: '-p 100'
 #   paths:
 #     drush-script: '/path/to/drush'
+#   env-vars:
+#     PATH: /bin:/usr/bin:${env.HOME}/.composer/vendor/bin
```

Comment:
Using `env.home` is ambiguous here. Some folks will think it refers to the remote system but it I think it refers to the local. We need to document or remove.

Reply:
(trying github suggestion feature...)
```suggestion
#     PATH: /bin:/usr/bin:/home/www-admin/.composer/vendor/bin
```

Reply:
Yes that part was plain wrong. Although i think that `$HOME` for the remote env var might work and make sense here, let's omit that until it's tested.